presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoRpmController.java

```diff
@@ -152,12 +156,14 @@ public Response restartUsingRpm()
      */
     public Response statusUsingRpm()
     {
+        String prestoVersion;
         try {
             if (!isInstalled()) {
                 LOGGER.info("Presto is not installed");
-                return Response.status(OK).entity(GSON.toJson("Presto is not installed"))
+                return Response.status(OK).entity(GSON.toJson(ImmutableMap.of("installed", false)))
                         .type(APPLICATION_JSON).build();
             }
+            prestoVersion = getVersion();
         }
         catch (PrestoManagerException e) {
             LOGGER.error(e.getCause(), e.getMessage());
```

Comment:
On the line after this, don't convert the error entity to JSON (which won't happen until you rebase onto master again).